libyul/AssemblyStack.h

```diff
@@ -39,6 +39,13 @@ namespace solidity::langutil
 class Scanner;
 }
 
+namespace solidity::evmasm
```

Comment:
This is not needed in the header, is it?

Reply:
It is not needed as I moved functionality to AssemblyItems, so, removed.